### PR TITLE
Add ~ubuntu-server package subscription process

### PR DESCRIPTION
**Pull request by basak**

### CODE REVIEW

File: MainInclusion.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Further information and rationale
+
+In the future, we might be able to enrich the subscription data further, such as to indicate specific releases that we support for particular packages. This becomes relevant when we change support status for a package. For example, after we have decided to stop supporting a particular package in main, moving it to universe, we must still retain the subscription until the final release in which the package was supported reaches EOL.
+
+We use `~canonical-server` as the owner for this repository because:
+
+ * We have automation driven from the data, which means we must be able to trust the input
+ * `~ubuntu-server`, which would be the logical place to keep data about `~ubuntu-server` subscriptions, has an open membership policy, which means anybody on the Internet might be able to subvert our script if we had stored it there
+ * Ultimately, Canonical have the authority in Ubuntu to decide what is and isn't in main, so restricting changes to Canonical staff makes sense
```
Comment (utkarsh2102) [inline MainInclusion.md:50]:
```suggestion
 * Ultimately, Canonical has the authority in Ubuntu to decide what is and isn't in main, so restricting changes to Canonical staff makes sense
```

File: MainInclusion.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ * We don't perceive a need to restrict maintenance to only uploaders
```
Comment (utkarsh2102) [inline MainInclusion.md:51]: Probably also worth adding fullstops at the end, but that's v trivial. Heh.

**Review by utkarsh2102 — COMMENTED** [review]
In general, I like it. We can add on improvements later but this is good enough as-is. However, one thing, I am worried about is the line length. 😞 

All the lines are awfully long, but no problem, I think I'll do the formatting of the entire handbook all at once. So maybe we can have this worked out later.

File: MainInclusion.md (hunk 1 above)
Comment (basak) [reply MainInclusion.md:50]: This is British English usage. I'm British. Therefore I'm allowed to use the plural :-)

https://english.stackexchange.com/questions/1338/are-collective-nouns-always-plural-or-are-certain-ones-singular

**Comment by basak** [comment]
> All the lines are awfully long, but no problem, I think I'll do the formatting of the entire handbook all at once.

I was just following the pattern set in most of the other files. Only a few are wrapped; most aren't. But yes - we should decide what to do consistently, and then do that.

File: MainInclusion.md (hunk 1 above)
Comment (utkarsh2102) [reply MainInclusion.md:50]: Ooh, this is a really good TIL moment for me. Thanks, definitely. All well. However, I'd have liked the consistency throughout the handbook, but all good. :)

**Comment by utkarsh2102** [comment]
> I was just following the pattern set in most of the other files. Only a few are wrapped; most aren't. But yes - we should decide what to do consistently, and then do that.

Yep, I'm waiting on the remaining PRs to get merged (or closed) and then I'll do the branch renaming (to `main`) and fix up the handbook (proofreading, changing to later releases, wrapping, et al.). In my TODO.

**Review by utkarsh2102 — APPROVED** [review]
K, thanks for fixing the fulllstop thingy. Approving and merging, follow-up fixes can happen anytime, as I said in the initial review. This looks good, thank you!

**Pull request merged**